real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -347,6 +347,46 @@ int8_t steering_center() {
     return 0;
 }
 
+void indicator_light_init()
+{
+    // set all pins to zero output
+    RX_STATUS_LIGHT_PORT &= ~_BV(3);
+    RX_STATUS_LIGHT_DDR |= _BV(3);
+    RX_STATUS_LIGHT_PORT &= ~_BV(5);
+    RX_STATUS_LIGHT_DDR |= _BV(5);
+    RX_STATUS_LIGHT_PORT_RED &= ~_BV(5);
+    RX_STATUS_LIGHT_DDR_RED |= _BV(5);
+}
+
```

Comment:
The functions to turn on the lights are never called.  Please add the function calls to the appropriate locations in the main loop.